packages/contracts-bedrock/contracts/test/BenchmarkTest.t.sol

```diff
@@ -141,28 +149,46 @@ contract GasBenchMark_L1CrossDomainMessenger is Messenger_Initializer {
 }
 
 contract GasBenchMark_L1StandardBridge_Deposit is Bridge_Initializer {
+    uint128 INITIAL_BASE_FEE;
+
     function setUp() public virtual override {
         super.setUp();
+        INITIAL_BASE_FEE = op.INITIAL_BASE_FEE();
         deal(address(L1Token), alice, 100000, true);
         vm.startPrank(alice, alice);
+        L1Token.approve(address(L1Bridge), type(uint256).max);
     }
 
     function test_depositETH_benchmark_0() external {
+        setPrevBaseFee(vm, address(op), INITIAL_BASE_FEE);
         L1Bridge.depositETH{ value: 500 }(50000, hex"");
     }
 
     function test_depositETH_benchmark_1() external {
-        setPrevBaseFee(vm, address(op), INITIAL_BASE_FEE);
+        setPrevBaseFee(vm, address(op), 10 gwei);
         L1Bridge.depositETH{ value: 500 }(50000, hex"");
     }
 
     function test_depositERC20_benchmark_0() external {
-        L1Bridge.depositETH{ value: 500 }(50000, hex"");
+        setPrevBaseFee(vm, address(op), INITIAL_BASE_FEE);
```

Comment:
Can we use the `pauseGasMetering` and `resumeGasMetering` cheatcodes here to get a more accurate figure?

https://github.com/foundry-rs/forge-std/blob/master/src/Vm.sol#L290-L293